src/components/document-metadata-modal.yml

```diff
@@ -40,3 +52,6 @@ verifiedInfo:
     As maintainer, you can confirm that the document meets the quality requirements. Verified documents have higher relevance in the search results.
   de:
     Als Maintainer kannst du bestätigen, dass das Dokument den Qualitätsanforderungen entspricht. Überprüfte Dokumente haben eine höhere Relevanz auf der Suchseite.
+allowedOpenContributionInfo:
+  en: You can restrict the contribution that the open community can bring to this document.
+  de: You can restrict the contribution that the open community can bring to this document.
```

Comment:
Maybe: "Sie können die Teile des Dokuments einschränken, zu denen die offene Community beitragen kann." (in english that would be: "You can restrict the parts of the document the open community can contribute to.")

Reply:
Sure thing, with the difference of "**Du kannst** die Teile des Dokuments einschränken, zu denen die offene Community beitragen kann.". Done!

Reply:
Of course "Du" :+1: